experiments.md

```diff
@@ -2,7 +2,15 @@
 
 ## Data Generation
 
-TODO: Generate TPC-H
+### Generate Locally (Optional)
+TPC-H (Scale 100, Partition 100)
+```bash
+docker build -t deepola-wake:sigmod2023-data -f dockerfiles/data.Dockerfile .
+docker run -i -v ./resources:/dataset/resources:rw -t deepola-wake:sigmod2023-data
+./data-gen.sh 100 100 /dataset
```

Comment:
You could do `docker run -rm ... <command>` so lines 9-10 can be combined into one liner.